jlm/hls/backend/rhls2firrtl/RhlsToFirrtlConverter.cpp

```diff
@@ -4016,7 +4017,11 @@ RhlsToFirrtlConverter::WriteCircuitToFile(const circt::firrtl::CircuitOp circuit
   std::string fileName = name + extension();
   std::error_code EC;
   ::llvm::raw_fd_ostream output(fileName, EC);
-  auto status = circt::firrtl::exportFIRFile(module, output);
+  auto status = circt::firrtl::exportFIRFile(
+      module,
+      output,
+      100,
```

Comment:
Could we assign this magic number to a variable with a descriptive name?

Reply:
Changed to size_t targetLineLength = 100;